Чекалина Виктория БИВТ-24-16

Vikvins opened the pull request.

NemirovichMark committed:
Add files via upload

NemirovichMark committed:
Add files via upload

NeAlyssa committed:
Merge pull request #14 from NemirovichMark/main

Upd Lab3

NemirovichMark committed:
Add files via upload

NeAlyssa committed:
Merge pull request #16 from NemirovichMark/main

Upd Lab3

NemirovichMark committed:
Update ProgramTests.cs

NeAlyssa committed:
Merge pull request #17 from NemirovichMark/main

Update ProgramTests.cs

Vikvins committed:
Чекалина Виктория БИВТ-24-16

NeAlyssa commented:
В 2_20 было убрано зануление суммы и сделано под условие из методички. Сумма по четным индексам считается только, если отрицательный элемент находится до минимума. В остальных случаях (отсутствует, расположен после, является им) считается сумма по нечетным. Обновите тесты и доработайте решение, остальное хорошо.

Vikvins committed:
Add files via upload

У меня по тестам во втором тесте был ответ "0" и не проходил тест с суммой нечётных элементов, поэтому занулила сумму. Только сейчас увидела обновлённые тесты.

NeAlyssa commented:
Принято. Не забудьте загрузить работу в задание на Moodle для выставления баллов за допуск.
